airflow-core/src/airflow/ui/src/i18n/locales/zh-TW/common.json

```
@@ -17,19 +17,44 @@
     "auditLog": "審計日誌",
     "xcoms": "XComs"
   },
+  "createdAssetEvent_one": "已建立資源事件",
+  "createdAssetEvent_other": "已建立資源事件",
   "dag_one": "Dag",
   "dag_other": "Dags",
   "dagDetails": {
+    "catchup": "自動回填",
+    "concurrency": "同時執行數",
```

```suggestion
    "concurrency": "並行數",
```

let's use what you suggested 👍

---

Sure!